sdk/monitor/Azure.Monitor.OpenTelemetry/src/AzureMonitorOpenTelemetryExtensions.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Monitor.OpenTelemetry.Exporter;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+
+namespace Azure.Monitor.OpenTelemetry
+{
+    /// <summary>
+    /// Extension methods for setting up Azure Monitor OpenTelemetry in an <see cref="IServiceCollection" />.
+    /// </summary>
+    public static class AzureMonitorOpenTelemetryExtensions
+    {
+        /// <summary>
+        /// Adds Azure Monitor OpenTelemetry into service collection.
+        /// </summary>
+        /// <param name="services"><see cref="IServiceCollection"/>.</param>
+        /// <param name="configuration"><see cref="IConfiguration"/>.</param>
+        /// <returns><see cref="IServiceCollection"/></returns>
+        public static IServiceCollection AddAzureMonitorOpenTelemetry(this IServiceCollection services, IConfiguration configuration)
+        {
+            var options = new AzureMonitorOpenTelemetryOptions();
+            configuration.Bind(options);
+            return services.AddAzureMonitorOpenTelemetry(options);
+        }
+
+        /// <summary>
+        /// Adds Azure Monitor OpenTelemetry into service collection.
+        /// </summary>
+        /// <param name="services"><see cref="IServiceCollection"/>.</param>
+        /// <param name="options">The <see cref="AzureMonitorOpenTelemetryOptions" /> instance for configuration.</param>
+        /// <returns><see cref="IServiceCollection"/></returns>
+        public static IServiceCollection AddAzureMonitorOpenTelemetry(this IServiceCollection services, AzureMonitorOpenTelemetryOptions? options = null)
+        {
+            options ??= new AzureMonitorOpenTelemetryOptions();
```

Comment:
TODO: We need to add support for `APPLICATIONINSIGHTS_CONNECTION_STRING` here. As a application insights SDK user I might assume it will work.

Reply:
Yes. This should be a part of exporter.

Reply:
or both may be? we are passing the connection string explicitly here.

Reply:
Modified the code, now it is inherited from AzureMonitorExporterOptions.